packages/garbo/src/tasks/barfTurn.ts

```diff
@@ -510,6 +510,28 @@ const NonBarfTurnTasks: AlternateTask[] = [
   },
 ];
 
+const guaranteedBullseye = () =>
+  get("everfullDartPerks").includes("25% Better bullseye targeting") &&
+  get("everfullDartPerks").includes("25% More Accurate bullseye targeting") &&
+  get("everfullDartPerks").includes("25% better chance to hit bullseyes");
+
+const safeToAttemptBullseye = () =>
+  guaranteedBullseye()
+    ? have($item`Everfull Dart Holster`)
+    : have($item`Everfull Dart Holster`) && have($item`spring shoes`);
+
+const canBullseye = () =>
+  guaranteedBullseye()
+    ? have($effect`Everything Looks Red`)
+    : have($effect`Everything Looks Red`) ||
+      have($effect`Everything Looks Green`);
+
+const dartLevelAtWhichThingsJustStartDying = 5;
```

Comment:
we name constants like this in `SCREAMING_SNAKE_CASE`, and I'm confident there's a more succinct name you can use here